### DISCUSSION TITLE
Problem Port forwarding

### DISCUSSION THREAD

**Discussion by acardipane** [Ysurac/openmptcprouter#2569] [Q&A]
in order not to have file transfer problems in transfer file rdp, filezilla, ftp, winscp I had to deactivate the vpn leaving only the proxy activated "shadowsocks", Everything now works correctly but portforwarding even if configured does not work. some idea ?

**Comment by Ysurac** [comment]
Port forwarding and anything that is not TCP use the VPN by default.

**Reply by acardipane** [reply]
hello, if I leave the vpn active (any) if you do a file transfer type copy paste rdp the speed is very very slow, same thing in port 21 filezilla / winscp.

**Reply by Ysurac** [reply]
Should not be the case, but you can try another VPN in the wizard or bypass the server IP.
I don't use rdp or ftp, in most case I use SSH/SFTP/SCP.

**Reply by acardipane** [reply]
hello, I tried with all the vpn  that are there by default. to bypass the ip what do you mean? thanks

**Reply by Ysurac** [reply]
I mean add destination IP in Services->OMR-Bypass.

**Reply by acardipane** [reply]
hi, ok, in this way, however, they will not use bonding (any connection) but only one internet connection, right?

**Reply by Ysurac** [reply]
True.
You can also check that you don't have any conflict with the VPN, VPN use IPs in 10.255.*.*, or that your VPS provider or ISP is not limiting speed when a VPN is detected.

**Reply by acardipane** [reply]
THANKS

**Reply by acardipane** [reply]
> True. You can also check that you don't have any conflict with the VPN, VPN use IPs in 10.255._._, or that your VPS provider or ISP is not limiting speed when a VPN is detected.

HI, vps does not limit speed. from the tests performed the problem is on RDP / FTP transfers.

**Reply by Ysurac** [reply]
RDP/FTP use TCP so only the proxy, the VPN doesn't do anything in this case. In FTP you use passive mode only (active shouldn't work) ?
